### PR TITLE
Update text classification- bug fix

### PR DESCRIPTION
**Pull request by VahidooX**

### CODE REVIEW

File: tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb [hunk 1]
```diff
@@ -187,6 +174,7 @@
         "os.makedirs(WORK_DIR, exist_ok=True)\n",
         "os.makedirs(DATA_DIR, exist_ok=True)\n",
         "\n",
+        "! wget \"https://firebasestorage.googleapis.com/v0/b/mtl-sentence-representations.appspot.com/o/data%2FSST-2.zip?alt=media&token=aabc5f6b-e466-44a2-b9b4-cf6337f84ac8\" --output-document SST-2.zip\n",
```
Comment (titu1994) [inline tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb:177]: If we are downloading data, do we need to download its license too (usually thats the case with most datasets)

File: tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb [hunk 2]
```diff
@@ -712,12 +700,19 @@
         "colab": {}
       },
       "source": [
+        "# move the model to the desire device for inference\n",
```
Comment (titu1994) [inline tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb:703]: to the "desired" device

**Review by titu1994 — COMMENTED** [review]
LGTM, just have a question regarding license

File: tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb (hunk 1 above)
Comment (VahidooX) [reply tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb:177]: It should not be there actually. Dopped it.

File: tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb (hunk 2 above)
Comment (VahidooX) [reply tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb]: fixed.

**Review by titu1994 — APPROVED** [review]
LGTM

**Pull request merged**